<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>539</width>
    <height>635</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>10</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>现场快递柜状态采集和控制系统</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>当前温度</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>50</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>设定温度</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>50</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>控制机状态</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="set_temperature">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>80</y>
     <width>113</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>11</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="open_drawer">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 246, 194);
</string>
   </property>
   <property name="text">
    <string>开抽屉</string>
   </property>
  </widget>
  <widget class="QPushButton" name="control_temperature">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>250</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 246, 194);</string>
   </property>
   <property name="text">
    <string>启停温度控制</string>
   </property>
  </widget>
  <widget class="QPushButton" name="set_contemper">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 246, 194);</string>
   </property>
   <property name="text">
    <string>设置控制温度</string>
   </property>
  </widget>
  <widget class="QPushButton" name="set_sys">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>250</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 246, 194);</string>
   </property>
   <property name="text">
    <string>设置系统参数</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>440</y>
     <width>281</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="text">
    <string>抽屉的状态</string>
   </property>
  </widget>
  <widget class="QPushButton" name="openbutton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>350</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(85, 170, 255);</string>
   </property>
   <property name="text">
    <string>开机</string>
   </property>
  </widget>
  <widget class="QPushButton" name="closebutton">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>350</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(85, 170, 127);</string>
   </property>
   <property name="text">
    <string>关机</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="chose_drawer">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>170</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>170</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>选择抽屉</string>
   </property>
  </widget>
  <widget class="QPushButton" name="close_drawer">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 246, 194);
</string>
   </property>
   <property name="text">
    <string>关抽屉</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="controller_state">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>80</y>
     <width>113</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>11</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="now_temperature">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>113</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>11</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="dr10">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>550</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>抽屉十</string>
   </property>
  </widget>
  <widget class="QPushButton" name="dr3">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>470</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>抽屉三</string>
   </property>
  </widget>
  <widget class="QPushButton" name="dr8">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>550</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>抽屉八</string>
   </property>
  </widget>
  <widget class="QPushButton" name="dr1">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>470</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>抽屉一</string>
   </property>
  </widget>
  <widget class="QPushButton" name="dr9">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>550</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>抽屉九</string>
   </property>
  </widget>
  <widget class="QPushButton" name="dr2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>470</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>抽屉二</string>
   </property>
  </widget>
  <widget class="QPushButton" name="dr4">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>470</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>抽屉四</string>
   </property>
  </widget>
  <widget class="QPushButton" name="dr7">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>550</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>抽屉七</string>
   </property>
  </widget>
  <widget class="QPushButton" name="dr6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>550</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>抽屉六</string>
   </property>
  </widget>
  <widget class="QPushButton" name="dr5">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>470</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>抽屉五</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>140</y>
     <width>321</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>PS：抽屉状态，黄色为开，绿色为关</string>
   </property>
  </widget>
  <widget class="QPushButton" name="temper_chart">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>380</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 246, 194);</string>
   </property>
   <property name="text">
    <string>查看温度曲线</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
